### DISCUSSION TITLE
Integration with PetLibro PLAF103 V2

### DISCUSSION THREAD

**Discussion by LilTrublMakr** [make-all/tuya-local#864] [Q&A]
I recently got the PetLibro PLAF103 V2 and wanted to integrate it in to Home Assistant. I see that it is possible through the Tuya integration. I am unable to get the feeder to pair with the Tuya app. It seems very similar to the PetLibro Lite app but I was not able to get the feeder to connect to that app either. I had to use the PetLibro app which makes me think that they changed the protocol or something like that,

Has anyone been able to integrate the V2 to Home Assistant?

**Comment by PeckhamHead** [comment]
I did have a PetLibro PLAF103 V1. It was officially PetLibro Lite App (looks like Tuya). I was able to pair it with standard Tuya. Tuya allowed Alexa and HA integration. It worked really well.
Due to a bug with the buttons on the device, I had to send it back.

The replacement is a PetLibro PLAF103 V2. Officially Petlibro app, this app doesn't look like Tuya. Tuya app does not pair with the V2.
Although the device works great through its own app, I cannot integrate with HA or Alexa.

FYI - I will do some network investigation in time and try to determine what it is communicating with, but for the moment I cannot get this into HA.

**Reply by PeckhamHead** [reply]
I did some packet capture. The PetLibro PLAF103 V2 connects to an AWS IP in the US using MQTT..
Destination Name:    ec2-44-204-136-70.compute-1.amazonaws.com
Address:  44.204.136.70
Port 1883

Not sure that helps
